.gitignore

```diff
@@ -5,7 +5,7 @@ lib/
 dist/
 .cache/
 .npmrc
-.idea/
+yarn.lock
```

Comment:
We should be using yarn so I don't want to be ignoring its lock file

Reply:
Sure, have removed this